app-web/src/components/Search/SearchPills.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import shortid from 'shortid';
+import Pill from '../UI/Pill';
+import styled from '@emotion/styled';
+
+const ClearPill = styled(Pill)`
+  background-color: #fff;
+  color: #1b64a0;
+`;
+
+const PillContainer = styled.div`
+  display: flex;
+  flex-flow: row wrap;
+`;
+/**
+ * Search Pills Component
+ * @param {Object} props
+ * @param {Func} props.onDelete
+ * @param {Array} props.query array of search tokens ['token', 'token']
+ */
+export const SearchPills = ({ onDelete, query, showClear, onClear }) => {
+  // if no query or if the query is a whitespace (when search is pressed without typing)
+  if (query.length === 0 || (query.length === 1 && query[0] === '')) {
+    return null;
+  }
+  const pills = [
```

Comment:
for now this is an array of one item. this will be moved to be a dynamic list once queries are tokenized in future prs.